docs/schemas/raw/lambda_config.yaml

```diff
@@ -22,12 +22,6 @@ properties:
       How long, in minutes, to cache config loaded from the config_table
     maximum: 10000
     minimum: 0
-  cookie_ttl:
```

Comment:
I'm not sure it's the best idea to immediately remove these from the schema.  That will cause any existing config files to be considered invalid and will fail deployment.  At the very least, it means an update to exodus-cdn-playbooks will have to be deployed *exactly* with this code update - no earlier and no later - because old config won't be accepted by the new schema and vice-versa.

You will probably find that deployment goes more smoothly if you allow these to stay in the schema for now, even if the code no longer uses these config elements.  They can be cleaned up after everything is fully deployed and exodus-cdn-playbooks no longer tries to set these.

I think you'd want to let them stay in "properties" (which means putting them in config will be permitted), but remove them from "required" (which means omitting them in config is also permitted).  That'll give one version of the schema/code which supports both the currently deployed and proposed new config files.

Reply:
Restored docs and configuration file template, removed cookie-related settings from required list
Once gw is deployed and config updated a follow-up can be created to remove these